engine/sim_trajectory.py

```diff
@@ -73,10 +74,9 @@ def get_plot_boundaries(nodes, delta):
             (CSV_PATH + '/rpi_to_gui_simulation.csv'), "a")
 
     # ser = serial.Serial("/dev/cu.usbserial-017543DC", 57600) # RPI_GUI_TEST
-    r2d2 = Robot(0, 0, math.pi / 4, epsilon=0.2, max_v=0.5,
-                 radius=0.2, init_phase=Phase.TRAVERSE, 
-                 is_sim=is_sim, is_store=is_store)
-
+    r2d2_state = Robot_State(x_pos=0, y_pos=0, heading=math.pi / 4, epsilon=0.2, max_velocity=0.5, radius=0.2)
```

Comment:
the initial heading should probably be read by the imu? why is this pi?

Reply:
Hm not sure 😅 , but just reiterating that the purpose of this PR was to move the attributes out from Robot to Robot_State, so changes to other files are for the purpose of updating how initialization of Robot to match this refactoring (passing in Robot_State and moving initialization there instead).

It's a good point to question why we have been initializing things this way, just a separate issue.

Reply:
maybe bc sim so can't get it by reading imu bc no imu in sim?